service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -24,8 +24,8 @@
 public abstract class AbstractEmailContentProvider {
 
     final String uiBaseUrl;
```

Comment:
why changes to this file?

Reply:
in order to reuse instead of doing a `super.uiBaseUrl` when building personalisations for the separate email templates as we do not need all values from `getCasePersonalisationBuilder(...)`